### PR TITLE
Update Campaign cards on Homepage

### PR DESCRIPTION
**Pull request by ani-kalpachka**

### CODE REVIEW

File: src/components/client/index/sections/ActiveCampaignsSection/ActiveCampaignCard/ActiveCampaignCard.tsx [hunk 1]
```diff
@@ -63,19 +47,35 @@ export default function ActiveCampaignCard({ campaign, index }: Props) {
             : targetAmount.split('.')[0]}
         </Sum>
       </SumWrapper>
+      <Grid
+        sx={{
+          background: `url(${campaignImagesUrl})`,
```
Comment (sashko9807) [inline src/components/client/index/sections/ActiveCampaignsSection/ActiveCampaignCard/ActiveCampaignCard.tsx:37]: Not that it is going to be issue for now, but it is better to load images through Next's `<Image>` component when possible, as it offers some in-built image optimizations, which might result in slight performance improvement. This is just something to keep in mind. 

Leaving the necessary docs to read if you are interested.
[Next.js - Image Component](https://nextjs.org/docs/pages/api-reference/components/image)
[Next.js - Image Optimization](https://nextjs.org/docs/pages/building-your-application/optimizing/images)

File: src/components/client/index/sections/ActiveCampaignsSection/ActiveCampaignCard/ActiveCampaignCard.tsx [hunk 2]
```diff
@@ -63,19 +47,35 @@ export default function ActiveCampaignCard({ campaign, index }: Props) {
...
+          height: '100%',
+          backgroundSize: 'cover',
```
Comment (sashko9807) [inline src/components/client/index/sections/ActiveCampaignsSection/ActiveCampaignCard/ActiveCampaignCard.tsx:39]: Won't this approach cause some images to be either over-stretched or cut from the frame? I think a  better approach would be to set the width to `100%` and then use the [aspectRatio CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio), to adjust the image's height.

**Review by igoychev — APPROVED** [review]

File: src/components/client/index/sections/ActiveCampaignsSection/ActiveCampaignCard/ActiveCampaignCard.tsx (hunk 1 above)
Comment (igoychev) [reply src/components/client/index/sections/ActiveCampaignsSection/ActiveCampaignCard/ActiveCampaignCard.tsx:37]: Great catch @sashko9807 ! We talked with @ani-kalpachka to make the image changes in separate PR to not delay the already ready visual improvements.

Comment (ani-kalpachka) [reply src/components/client/index/sections/ActiveCampaignsSection/ActiveCampaignCard/ActiveCampaignCard.tsx:37]: Thanks for the review! 
I will apply this change in a separate PR.

**Pull request merged**